examples/ui/text.rs

```diff
@@ -41,7 +41,7 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
                 color: Color::WHITE,
             },
         ) // Set the alignment of the Text
-        .with_text_alignment(TextAlignment::TOP_CENTER)
+        .with_text_alignment(TextAlignment::Center)
```

Comment:
Same here, this needs a corresponding change to its parent style to ensure it still ends up being top-center

Reply:
This is an example of why VerticalAlign needs to be removed. TextAlignment::TOP_CENTER is identical to TextAlignment::CENTER.